docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -0,0 +1,100 @@
+---
+
+A Data Asset is a collection of records that you create when you connect to your Data Source. When you connect to your Data Source, you define a minimum of one Data Asset. You use these Data Assets to create the Batch Requests that define the data that is provided to your Expectations.
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud).
+
+- You have [set up your environment](../set_up_gx_cloud.md) including setting the `GX_CLOUD_SNOWFLAKE_PASSWORD` environment variable. 
+
+- You have a [Snowflake account](https://docs.snowflake.com/en/user-guide-admin) with `ACCOUNTADMIN` access.
+
+
+## Create a Data Asset
+
+Create a Data Asset to define the data you want GX Cloud to access. Currently, the GX Cloud user interface is configured for Snowflake. To connect to Data Assets on another Data Source, see [Connect to source data](https://deploy-preview-8760.docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_lp) in the GX OSS documentation. 
+
+1. In GX Cloud, click **Data Assets** > **New Asset**.
+
+2. Complete the following mandatory fields:
+
+    - **Datasource name**: Enter a meaningful name for the Data Asset.
+
+    - **Username**: Enter your Snowflake username.
+
+    - **Password variable**: Enter `GX_CLOUD_SNOWFLAKE_PASSWORD`.
+
+    - **Account or locator**: Enter your Snowflake account or locator information. The locator value must include the geographical region. For example, `us-east-1`. To locate these values see [Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier).
+
+3. Optional. Complete the following fields:
+
+    - **Database**: Enter the name of the Snowflake database where the data you want to validate is stored.
+ 
+    - **Schema**: Enter the name of the schema for the Snowflake database where the data you want to validate is stored.
+
+    - **Warehouse**: Enter the name of the Snowflake database warehouse.
+
+    - **Role**: Enter your Snowflake role.
+
+    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+
+4. Optional. Clear **Create temp table** if you don't want to create a temporary database table.
+
+5. Optional. Clear **Test connection** if you don't want to test the Data Asset connection.
+
+6. Select **Table Asset** or **Query Asset** and complete the following fields:
+
+    - **Asset name**: Enter a name for the Data Asset.
+
+    - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+
+    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
+
+7. Optional. Select **Add table/query** to add additional tables or queries and repeat step 6.
+
+8. Click **Finish**.
+
+9. Create an Expectation. See [Create an Expectation](/docs/cloud/expectations/manage_expectations#create-an-expectation).
+
+## Edit a Data Asset
```

Comment:
Do we need an introductory statement saying editing can't be done via the web ui currently? Jumping straight from a UI based workflow to a notebook one seems a little abrupt.

Reply:
@billdirks I am documenting how users can complete tasks **_currently_**. If they need to jump to a Jupyter Notebook, that's what they are instructed to do. This is the approach that Hernan and others requested. As you discovered, the gaps in functionality are obvious when the process is documented.

Reply:
I meant I thinkwe should be explicit that this functionality doesn't not that we should leave this out. That is, have an intro sentence saying: "We do not currently support editing expectations in the UI." before jumping into the jupyter notebook steps.